images/aws/terraform.tf

```diff
@@ -91,6 +91,12 @@ resource "aws_sqs_queue" "integration_queue" {
   content_based_deduplication   = true
 }
 
+resource "aws_sqs_queue" "sqs_queue" {
+  name                          = "egress-queue.fifo"
+  fifo_queue                    = true
```

Comment:
Is it a fifo queue in real life? If so cool, otherwise we should not test on one here.